app/policies/user_policy.rb

```diff
@@ -7,6 +7,7 @@ def new?
   alias_method :index?, :new?
 
   def edit?
+    return unless current_user
```

Comment:
In what situation was `current_user` not set here? Is it worth addressing the root problem there?